### PR TITLE
#1683 Capitalized SABO under the RR status names in the Finance page

### LINKED ISSUE #1683: Fix the status name spelling in the Finance page
### Description

We currently have `Sabo` in one of the RR status names, 'Submitted to Sabo' with lowercase letters. We want to capitalize all letters in `Sabo` so we get `SABO`.

image

### Acceptance Criteria

Change the word `Sabo` to `SABO` in one of the RR status names, `Submitted to Sabo`

### Proposed Solution

Search where we defined RR status names in `FinishLine\src\frontend\src\pages\FinancePage\ReimbursementRequestsSection.tsx` and change the `Sabo` to `SABO` from the status name, 'Submitted to Sabo.'

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by christian-bacalhau**
## Changes

Capitalized all the letters in SABO under the RR status names on the finance page so it reads "Submitted to SABO"

## Screenshots

Screenshot 2024-02-28 at 5 12 53 PM

Screenshot 2024-02-28 at 5 12 57 PM
Screenshot 2024-03-28 at 5 10 23 PM
Screenshot 2024-03-28 at 5 10 26 PM
41c">
Screenshot 2024-03-28 at 5 11 19 PM
Screenshot 2024-03-28 at 5 11 22 PM
Screenshot 2024-03-28 at 5 12 03 PM
Screenshot 2024-03-28 at 5 12 13 PM
Screenshot 2024-03-29 at 12 43 12 AM
Screenshot 2024-03-29 at 12 43 15 AM
Screenshot 2024-03-29 at 12 43 23 AM

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1683

### CODE REVIEW

**Review by RChandler234 — CHANGES_REQUESTED** [review]
Can you also update all the other places 'Sabo' is displayed on the frontend

**Review by callmejim1226 — APPROVED** [review]
Looks good to me! Nice work!

**Review by callmejim1226 — CHANGES_REQUESTED** [review]
There are still few frontend pages that has a label of `Sabo` text (ex. ReimbursementRequestDetailsView.tsx) Make sure to also replace al `Sabo` text to SABO

File: src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/AddSABONumberModal.tsx [hunk 1]
```diff
@@ -58,12 +58,12 @@ const AddSABONumberModal = ({ modalShow, onHide, reimbursementRequestId }: AddSA
...
       onFormSubmit={onSubmit}
       disabled={!isValid}
-      formId="add-sabo-number"
+      formId="add-SABO-number"
     >
       <FormControl>
         <ReactHookTextField
           control={control}
-          name="saboNumber"
+          name="SABONumber"
```
Comment (callmejim1226) [inline src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/AddSABONumberModal.tsx:66]: We don't have to capitalize the sabo text here since `formId `and `name `fields represent the component id and name. We only want to capitalize the sabo text that gets displayed on the webpage, so we can undo the changes in this file.

**Review by callmejim1226 — CHANGES_REQUESTED** [review]
Looks good! One last change for final review!

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥
